services/github_service.py

```diff
@@ -1170,14 +1170,23 @@ def modify_gha_minutes_quota_threshold(self, new_threshold):
             "https://api.github.com/repos/ministryofjustice/operations-engineering/actions/variables/GHA_MINUTES_QUOTA_THRESHOLD", json.dumps(payload), headers=headers)
 
     @retries_github_rate_limit_exception_at_next_reset_once
-    def get_gha_minutes_quota_threshold(self):
+    def _get_repository_variable(self, variable_name):
         actions_variable = self.github_client_core_api.get_repo(
-            'ministryofjustice/operations-engineering').get_variable("GHA_MINUTES_QUOTA_THRESHOLD")
-        return int(actions_variable.value)
+            'ministryofjustice/operations-engineering').get_variable(variable_name)
+        return actions_variable.value
+
+    def get_gha_minutes_quota_threshold(self):
```

Comment:
I can't see this function being called anywhere? Is it being used?

Reply:
@levgorbunov1 it is used inside the **check_if_gha_minutes_quota_is_low** .